lib/EndOfItem/tests/EndOfItem.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import '../../../test/jest/__mock__';
+
+import { renderWithIntl } from '../../../test/jest/helpers';
+import { EndOfItem } from '../EndOfItem';
+
+const setupEndOfItem = ({
+  title,
+  className,
+} = {}) => renderWithIntl(
+  <EndOfItem
+    title={title}
+    className={className}
+  />,
+  [{
+    prefix: 'stripes-components',
+    translations: { endOfList: 'stripes-components.endOfList' },
+  }]
+);
+
+describe('EndOfItem', () => {
+  const customClass = 'my-class';
+
+  describe('rendering EndOfItem with title', () => {
+    const title = 'Title';
+
+    it('should add custom class style to layout container', () => {
+      setupEndOfItem({
```

Comment:
Maybe it will be better to do mounting in beforeEach, considering passing `customClass` will not break other tests in this describe block

Reply:
Makes sense. Will do.